beets/importer.py

```diff
@@ -943,7 +943,9 @@ def is_archive(cls, path):
         if not os.path.isfile(path):
             return False
 
-        for path_test, _ in cls.handlers():
+        for path_test, x in cls.handlers():
+            if isinstance(path, bytes):
+                path = util.py3_path(path)
```

Comment:
i already committed it without the bytes check, and everything so far is in order.